Library/Formula/discount.rb

```diff
@@ -5,14 +5,20 @@ class Discount < Formula
   url 'http://www.pell.portland.or.us/~orc/Code/discount/discount-2.1.7.tar.bz2'
   sha1 '517bcf7409d8c02b3e57f51264b2e110f8a03120'
 
+  option "with-fenced-code", "Enable Pandoc-style fenced code blocks."
+
   conflicts_with 'markdown',
     :because => 'both discount and markdown ship a `markdown` executable.'
 
   def install
+    opts = []
+    opts << "--with-fenced-code" if build.include? "with-fenced-code"
```

Comment:
I'm sorry for wrong use.
Because this formula hasn't this configure option, I don't want to change current behavior.